<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewProjectDialog</class>
 <widget class="QDialog" name="NewProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>320</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>471</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>471</width>
    <height>320</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Create a new Project</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>290</y>
     <width>461</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditLocation">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonBrowse">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>258</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLocation">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>263</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Location:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditName">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>391</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelName">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>233</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>451</width>
     <height>211</height>
    </rect>
   </property>
   <property name="currentRow">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>Blank Project</string>
    </property>
    <property name="icon">
     <iconset resource="../../resources/icons.qrc">
      <normaloff>:/icons/application.png</normaloff>:/icons/application.png</iconset>
    </property>
   </item>
  </widget>
  <zorder>buttonBox</zorder>
  <zorder>lineEditLocation</zorder>
  <zorder>buttonBrowse</zorder>
  <zorder>labelLocation</zorder>
  <zorder>labelName</zorder>
  <zorder>listWidget</zorder>
  <zorder>lineEditName</zorder>
 </widget>
 <tabstops>
  <tabstop>lineEditName</tabstop>
  <tabstop>lineEditLocation</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>buttonBrowse</tabstop>
  <tabstop>listWidget</tabstop>
 </tabstops>
 <resources>
  <include location="../../resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
